### ISSUE TITLE
Open FancyPlace with a Mapping App

### ISSUE THREAD

**Issue by Greulty** [gabm/FancyPlaces#14]
This already has the local locations storage features I have wanted from other apps. I understand this will never be a navigation app. IIt would would be handy to long click a location location to location to export GPS coordinates coordinates to coordinates to the clipboard.

Please excuse the doubled words. The mobile keyboard keyboard does keyboarddoesthis keyboarddoesthiswheb keyboarddoesthiswheb exceed the cache of keystrokes and it is far too vexing to attempt extensive editing.

**Comment by gabm** [comment]
hey, i think you would like a "Shortcut" to open a certain fancy place with a mapping application (like locus,google maps etc), is that right?

well that's a great idea, basically i already had it implemented months ago but the feature got lost with the transition to osmdroid... I'll reimplement it :)

**Comment by k3b** [comment]
great idea

is it possible to support the geo-uri format?

```
  geo:52.123,9.321?q=52.123,9.321(fancy+name)&d=fancy+description&t=2015-02-10T08:04:45Z
```

it should be compatible with google maps for android and with LocationMapViewer

q = "query" containing the point name in "(...)" url-encoded
d = description text url-encoded
t = timestamp iso format

for details see https://github.com/k3b/LocationMapViewer/wiki/data#geo

**Comment by gabm** [comment]
I implemented this feature, it will be included in upcoming release v1.2.0 ... thank you both for your input...
